src/rules/no-duplicate-load-rules/index.js

```diff
@@ -0,0 +1,175 @@
+"use strict";
+
+const { utils } = require("stylelint");
+const valueParser = require("postcss-value-parser");
+const getAtRuleParams = require("../../utils/getAtRuleParams");
+const namespace = require("../../utils/namespace");
+const isPlainObject = require("is-plain-object");
+const { isBoolean } = require("../../utils/validateTypes");
+const ruleUrl = require("../../utils/ruleUrl");
+
+const ruleName = namespace("no-duplicate-load-rules");
+const messages = utils.ruleMessages(ruleName, {
+  rejected: duplicate => `Unexpected duplicate load rule ${duplicate}`
+});
+
+const meta = {
+  url: ruleUrl(ruleName)
+};
+
+function rule(primary, secondaryOptions) {
+  return (root, result) => {
+    const validOptions = utils.validateOptions(
+      result,
+      ruleName,
+      {
+        actual: primary
+      },
+      {
+        actual: secondaryOptions,
+        possible: {
+          includeForward: isBoolean,
```

Comment:
You mentioned that you removed the secondary options, but they are still listed here.